test/test_torch.py

```diff
@@ -11684,6 +11684,25 @@ def _test(sizes, n_fft, hop_length=None, win_length=None, win_sizes=None,
         _test((10,), 5, 4, win_sizes=(11,), expected_error=RuntimeError)
         _test((10,), 5, 4, win_sizes=(1, 1), expected_error=RuntimeError)
 
+    def test_fft_input_modification(self, device):
```

Comment:
could you add a ref to the issue here also?

Reply:
Done